ghostwriter-jdk-v7/src/main/java/io/ghostwriter/openjdk/v7/GhostWriterAnnotationProcessor.java

```diff
@@ -50,8 +50,7 @@ public boolean process(Set<? extends TypeElement> annotations, RoundEnvironment
             }
         }
 
-        final boolean doClaimAnnotations = false;
-        return doClaimAnnotations;
+        return false;
```

Comment:
A comment seems to be in order.
Why is it always "false", I wonder? Sounds like a FIXME actually.